api/query/search_test.go

```diff
@@ -27,7 +27,7 @@ func TestPrepareSearchResponse(t *testing.T) {
 	}
 	filters := shared.QueryFilter{
 		RunIDs: runIDs,
-		Q:      "/b/",
+		Q:      "/B/",
```

Comment:
I'd like to see one more test where the paths are capital but the query string is lowercase.